R/pkg/R/utils.R

```diff
@@ -750,7 +750,7 @@ launchScript <- function(script, combinedArgs, wait = FALSE, stdout = "", stderr
   if (.Platform$OS.type == "windows") {
     scriptWithArgs <- paste(script, combinedArgs, sep = " ")
     # on Windows, intern = F seems to mean output to the console. (documentation on this is missing)
-    shell(scriptWithArgs, translate = TRUE, wait = wait, intern = wait) # nolint
+    shell(scriptWithArgs, translate = TRUE, wait = wait, intern = wait)
```

Comment:
so we had `nolint` before but this change remove it?

Reply:
Yes. I verified that it doesn't work as you see in the previous GitHub run.
(Both online `nolint` and blocked `nolint start ~ nolint end`).
Instead, we exclude the root cause, rule `object_usage_linter`.

Reply:
This might work previously on Window environment. However, after this PR, this will pass for all Window/Linux/Mac environment.